server/api/stats.go

```diff
@@ -50,7 +55,43 @@ func (h *statsHandler) GetRegionStatus(w http.ResponseWriter, r *http.Request) {
 	if r.URL.Query().Has("count") {
 		stats = rc.GetRegionStatsCount([]byte(startKey), []byte(endKey))
 	} else {
-		stats = rc.GetRegionStatsByRange([]byte(startKey), []byte(endKey))
+		stats = rc.GetRegionStatsByRange([]byte(startKey), []byte(endKey), false)
 	}
 	h.rd.JSON(w, http.StatusOK, stats)
 }
+
+// @Tags	 distributions
+// @Summary  Get region stats and hot flow statistics of a specified range.
+// @Param    start_key  query  string  true   "Start key"
+// @Param    end_key    query  string  true   "End key"
+// @Param    engine     query  string  false  "Engine type such as  tikv or tiflash"
+// @Produce  json
+// @Success  200  {object}  statistics.RegionStats
+// @Router   /distributions/region [get]
+func (h *statsHandler) GetRegionDistributions(w http.ResponseWriter, r *http.Request) {
```

Comment:
It looks like this API is almost the same as GetRegionStatusByKeyRange, why not extend the existing API?

Reply:
That's fine, but this api is already complex. It needs to consider three scenes such as:
1. the informations of the regions.
2. only needs the region count for `spilt regions` SQL or Tiflash.
3. the informations of the regions includes the hot statistics